documentation/book/security.adoc

```diff
@@ -1,61 +1,75 @@
+`<cluster-name>-zookeeper-nodes`::
+Contains all the Zookeeper node private and public keys (certificates signed with the cluster CA).
 
 All the keys are 2048 bits in size and are valid for 365 days from initial generation.
 
 NOTE: "certificates rotation" for generating new ones on their expiration will be supported in future releases.
 
-=== Listeners
+=== Kafka Listeners
 
 The data encryption is provided on two different listeners exposed by each Kafka broker.
 
-Interbroker communication is through the `REPLICATION` listener on port 9091, which is encrypted by default.
+Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
 Encrypted communication with clients is provided through the `CLIENTTLS` listener on port 9093.
 
 NOTE: Un-encrypted communication with clients is still possible through the `CLIENT` listener on port 9092.
 
-=== Clients connection via TLS
+=== Kafka Client connections via TLS
 
-If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the client's CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
-The clients CA certificate can be extracted from the generated `<cluster-name>-kafka-cert` Secret with the following command if the Kafka cluster is running on {OpenShiftName}.
-
-[source,shell]
-oc get secret <cluster-name>-kafka-cert -o jsonpath='{.data.clients-ca\.crt}' | base64 -d > clients-ca.crt
+If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the cluster CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
+The cluster CA certificate can be extracted from the generated `<cluster-name>-cluster-ca-cert` Secret.
 
 ifdef::Kubernetes[]
-If the Kafka cluster is running on {KubernetesName}, the same result can be achieved with the following command.
+On {KubernetesName}, the certificate can be extracted with the following command:
 
 [source,shell]
-kubectl get secret <cluster-name>-kafka-cert -o jsonpath='{.data.clients-ca\.crt}' | base64 -d > clients-ca.crt
+kubectl get secret <cluster-name>-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
 
 endif::Kubernetes[]
-The native Kafka client, in order to use it, needs the certificate to be imported in a truststore using the `keytool` command line tool.
+
+On {OpenShiftName}, the certificate can be extracted with the following command:
+
+[source,shell]
+oc get secret <cluster-name>-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+
+The Kafka client will need to be configured to trust certificates signed by this CA.
+For the Java-based Kafka Producer, Consumer and Streams APIs, this is done by importing the CA certificate into the JVM's truststore using the keytool command, as follows:
```

Comment:
Minor rewrite to enhance the readability. Please change this para to "The Kafka client has to be configured to trust certificates signed by this CA. For the Java-based Kafka Producer, Consumer and Streams APIs, you can do this by importing the CA certificate into the JVM's truststore using the following keytool command: